calculator.py

```diff
@@ -1,11 +1,42 @@
 def addition(number_1, number_2):
+    return number_1 + number_2
 
 def subtraction(number_1, number_2):
+    return number_1 - number_2
 
 def multiplication(number_1, number_2):
+    return number_1 * number_2
 
 def division(number_1, number_2):
+    if(number_2 == 0):
+        return None
+    else :
+        return number_1 / number_2
 
 def get_input():
+    try : 
+        number_1 = float(input("Enter the first number : "))
+        operator = input("Enter the operation to be performed : ")
+        number_2 = float(input("Enter the second number : "))
+    except :
+        print("Enter a valid number/operator")
+    else :
+        if(operator == '+') :
+            addition_result = addition(number_1,number_2)
```

Comment:
do not use a new variable where not required **addition_result**,  **subtraction_result**,  **multiplication_result**
directly function can be called in print statement.

new variable is required for  **division_result** as you have some checks later.

Reply:
Okay mam